content/09.evolution.md

```diff
@@ -138,9 +97,95 @@ While it has been suggested that more thorough investigation of the origins of C
+
+The phylogeny of the species implicated in the origination of COVID-19 suggested that the host range of SARS-CoV-2 could encompass many species with a high level of interaction with humans.
+Humans last shared an ancestor with bats and pangolins almost 100 million years ago [@doi:10.1073/pnas.1702012114].
+Bats belong to the order Chiroptera and pangolins to Pholidota, which both belong to the clade _Pegasoferae_ [@doi:10.1016/j.cub.2013.09.014; @doi:10.1073/pnas.0603797103; @doi:10.1101/gr.5918807].
+They are closely related to many other species that have close relationships with humans, namely odd-toed ungulates (Euungulata) and carnivores (Carnivora) [@doi:10.1016/j.cub.2013.09.014; @doi:10.1073/pnas.0603797103; @doi:10.1101/gr.5918807; @url:http://www.timetree.org/public/data/pdf/Murphy2009Chap71.pdf].
+The part of the evolutionary tree that includes both humans and the Pegasoferae encompasses many species of significant social and economic importance. 
+Therefore, concerns were raised that the species with which humans have close interactions, many of which are much more closely related to bats and pangolins than humans are, could also be infected.
+It seemed plausible that the host range could include both livestock, many of which are odd-toed ungulates, and companion animals, many of which are carnivores.
+Infection of these animals was identified as a major concern [@doi:10.1089/vbz.2020.2650].
+
+Genomic analyses seeking to identify which species were likely to be susceptible focused largely on the comparative genetics of angiotensin-converting enzyme 2 (ACE2).
+ACE2 is the primary protein used by SARS-CoV-2 to enter the cell (see [@individual-pathogenesis]).
+Recognition of this protein is largely determined by domains in the S1 subunit of the RBD [@doi:10.1016/j.cell.2020.02.058].
+Alignment of the _ACE2_ sequence from 19 species revealed high conservation among mammals [@doi:10.1016/j.jinf.2020.02.013].
+This analysis suggested that non-human primates (three monkey and two ape species), companion animals (dogs and cats), and livestock (both odd- and even-toed ungulates) may all be susceptible to SARS-CoV-2 [@doi:10.1016/j.jinf.2020.02.013].
+Similarly, another study conducted an _in silico_ analysis of ACE2 protein structures and their predicted binding to SARS-CoV-2 for 410 vertebrate species [@doi:10.1073/pnas.2010146117].
+The species identified as having the highest predicted binding affinities were all primates, including humans.
+Other taxa with high predicted affinities included other primates, rodents, even-toed ungulates (namely, several species of cetaceans and deer), and anteaters.
+Reindeer were the only domesticated species predicted to belong to either of these groups, but many common zoo animal species with threatened or worse IUCN risk status were identified as at risk.
+
+Many cases of reverse zoonotic, or anthroponotic, SARS-CoV-2 transmission have been reported.
```

Comment:
```suggestion
Considering the evidence generated by _in silico_ studies, it may not be surprising that many cases of reverse zoonotic, or anthroponotic, SARS-CoV-2 transmission have been reported.
```